packages/node-core/src/utils/string.spec.ts

```diff
@@ -58,3 +58,25 @@ describe('handledStringify', () => {
     expect(result).toBe('Error without stack');
   });
 });
+
+describe('handledStringify depth', () => {
+  it('Different truncation situations at various depths', () => {
```

Comment:
Can we please expand this to include:
* Null values
* Undefined values
* A circular reference.